### PR TITLE
12-CORE-GET-api-articles-article_id

### PR DESCRIPTION
**Pull request by miors**
endpoint /api/articles/:article_id
added solution, test
updated API endpoints doc

### CODE REVIEW

File: __tests__/integration.test.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ describe("GET /api/articles/:article_id", () => {
         expect(article.article_img_url).toBe(
           "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
         );
+        expect(article.comment_count).toBe(11);
```
Comment (pippaamy) [inline __tests__/integration.test.js:64]: we really want this in a new test as we dont want to change previous tests :)

File: __tests__/integration.test.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/1")
       .expect(200)
       .then(({ body: { article } }) => {
-        expect(Object.keys(article)).toHaveLength(8);
+        expect(Object.keys(article)).toHaveLength(9);
```
Comment (pippaamy) [inline __tests__/integration.test.js:53]: the problem with using object.keys here is it makes the test brittle like here it now needs to be changed to pass! I wouldnt use it personally!

File: endpoints.json [hunk 3]
```diff
@@ -39,7 +39,8 @@
           "body": "I find this existence challenging",
           "created_at": "2020-07-09T20:11:00.000Z",
           "votes": 100,
-          "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+          "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          "comment_count": 11
```
Comment (pippaamy) [inline endpoints.json:43]: nicely remembered!

**Review by pippaamy — COMMENTED** [review]
Great job, we want to avoid changing tests as much as possible! I would test for it in a new test :)

File: __tests__/integration.test.js (hunk 2 above)
Comment (miors) [reply __tests__/integration.test.js:53]: removed

File: __tests__/integration.test.js (hunk 1 above)
Comment (miors) [reply __tests__/integration.test.js:64]: additional test created

**Pull request merged**